src/main/java/platform/codingnomads/co/springmvc/model/HomeController.java

```diff
@@ -18,6 +17,10 @@ public String index(Model model) {
         model.addAttribute("name", "Developer");
         model.addAttribute("language", "Java");
         model.addAttribute("framework", "Spring");
+        model.addAttribute("location", "Earth");
+        model.addAttribute("field", "Gold");
+        model.addAttribute("level", "4.7");
```

Comment:
It could be just a `4.7f` for the float number. 

I just to underline, that value could be of any type.

Reply:
Ok, so addAttribute works like JSON where numbers do not have quotes?

Reply:
It is a Java interface with the signature `addAttribute(String, Object)`, so the right part could be anything. 
You can also pass primitives, java compiler is smart enough to wrap them in the corresponding Object wrapper via a process called "Autoboxing and Unboxing".